appconfigservice/appconfig.yaml

```diff
@@ -91,6 +91,7 @@ paths:
         Auth: Requires a valid API Key for access.
       security:
         - ApiKeyAuth: []
+        - CoreTokenAuth: []
```

Comment:
Does this not provide easier access to all app configurations, especially as generating a Core access token is relatively straightforward when compared with the API key?

Reply:
Hi @sandeep-ps, this is actually a very good point. We will have to reconsider how API keys are being used and validated on the Core BB side. Thanks for catching this!

Reply:
Hi @sandeep-ps, I opened this issue https://github.com/rokwire/core-building-block/issues/244 to handle this issue on the Core BB. As you can see in the thread and related PR, we decided to validate the API key before allowing the user to generate a token at all. Once these Core BB changes have been approved and merged, I believe this issue will be resolved without needing any changes in this PR. This is because we can guarantee that anyone who has a valid token had a valid API key. Let me know if you see any issues with this approach. Thanks!